ci-operator/config/coreos/coreos-assembler/coreos-coreos-assembler-main.yaml

```diff
@@ -1,19 +1,45 @@
-binary_build_commands: make && make check
+base_images:
+  fedora:
+    name: fedora
+    namespace: coreos
+    tag: "34"
 build_root:
   image_stream_tag:
-    name: cosa-buildroot
+    name: fcos-buildroot
     namespace: coreos
-    tag: latest
+    tag: testing-devel
```

Comment:
I'm not sure I understand this, but it seems like it's trying to use `fcos-buildroot`, whereas it should just be independent from cosa. FCOS and cosa usually move to the next releasever at around the same time, though the former a bit earlier.

/me reads a bit of https://docs.ci.openshift.org/docs/architecture/ci-operator/

So, IIUC the model Prow really wants us to follow I think is the multi-stage build used in the majority of the OpenShift components. Whereas for cosa it's just really one Dockerfile. Maybe once we start shedding the build deps we could switch to that too. But for now, I think that implies we can drop `build_root` entirely. (E.g. we don't actually have a `binary_build_commands` anymore, so this wouldn't actually be used anyway if I understand right?)